### PR TITLE
Adds Istanbul for test coverage and change test script

### LINKED ISSUE #28: Figure out how to see test coverage
Need to figure out how to see test coverage in Mocha. Need to add unit tests of the models? They don't really have any methods other than the built in creation/deletion/update/associations/etc...

### PR DESCRIPTION
**Pull request by ryanmillergm**
**Adds Istanbul for test coverage:**
 - Installed Istanbul 
 - Changed test script to add Istanbul 
 
Resolves #28 
 
**The following checks have been completed:**
 - [x] Tested new feature(s) as well as any feasible edge cases
 - [x] Updated README for changes (new endpoints, new packages, etc)
 - [x] Merged in the latest master & resolved all merge conflicts
 - [x] Ran database migrations
 - [x] Ran the test suite - all tests are passing (or maybe skipped)
 - [ ] Checked affected endpoints in Postman

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "start": "node ./bin/www",
-    "test": "export NODE_ENV=test && mocha --recursive"
+    "test": "nyc mocha --recursive"
```
Comment (chakeresa) [inline package.json]:
```suggestion
    "test": "export NODE_ENV=test && nyc mocha --recursive"
```

Comment (chakeresa) [reply package.json]: We still need to specify that the tests are running in our test database (not development environment)

File: README.md [hunk 2]
```diff
@@ -16,6 +16,7 @@ View the project board at https://github.com/ryanmillergm/quantified_self/projec
  - Database: PostgreSQL v7.12.1
  - ORM: Sequelize v5.15.1
  - Testing: Mocha v6.2.0 & Chai v4.2.0
+ - Test Coverage: Istanbul v14.1.1
```
Comment (chakeresa) [inline README.md]:
```suggestion
 - Test Coverage: Istanbul (nyc) v14.1.1
```
Consider moving to "Other Packages" below -- not really a core part of our tech stack

**Comment by chakeresa** [comment]
Started new branch without the .nyc_output -- see other PR
